src/modules/client/components/contracts/ContractEndingModal.vue

```diff
@@ -62,6 +66,13 @@ export default {
         this.validations.otherMisc.$reset();
       }
     },
+    updateAndResetOtherMisc (event, prop) {
+      this.update(event, prop);
+      this.resetOtherMisc();
```

Comment:
tu peiux changer dans cette methode le delete -> le passer en this.update avec une string vide a priori

Reply:
done